src/backend/distributed/deparser/qualify_type_stmt.c

```diff
@@ -0,0 +1,160 @@
+/*-------------------------------------------------------------------------
+ *
+ * qualify_type_stmt.c
+ *	  Functins specialized in fully qualifying all type statements
+ *
```

Comment:
if we're not adding function comments, it would be good to give a more detailed description of what this file is for, in particular what does it mean to fully qualify a type statement?